src/libs/nav2_tasks/include/nav2_tasks/bt_action_node.hpp

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
All these BT specific files shouldn't go in the nav2_tasks folder should they? I thought the tasks were supposed to be independent of the implementation of the execution engine, whether it be BT, SMACH or whatever.